src/main/java/com/osiris/autoplug/client/tasks/SSHManager.java

```diff
@@ -0,0 +1,127 @@
+ */
+
+package com.osiris.autoplug.client.tasks;
+
+import java.io.IOException;
+
+import com.osiris.autoplug.client.configs.SSHConfig;
+import com.osiris.autoplug.client.network.online.connections.SSHServerConsoleReceive;
+import com.osiris.autoplug.client.network.online.connections.SSHServerSetup;
+import com.osiris.autoplug.client.utils.ConsoleOutputCapturer;
+import com.osiris.jlib.logger.AL;
+
+public class SSHManager {
+    private static Thread sshThread;
+    private static Thread consoleCaptureThread;
+    private static SSHServerSetup sshServerSetup;
+    private static SSHConfig sshConfig;
+    private static ConsoleOutputCapturer capturer;
+
+    static {
+        try {
+            sshConfig = new SSHConfig();
+            sshServerSetup = new SSHServerSetup();
+            capturer = new ConsoleOutputCapturer();
+            createThread();
+            createConsoleCaptureThread();
+        } catch (IOException e) {
+            AL.warn("Failed to initialize SSHManager", e);
+        } catch (Exception e) {
+            AL.warn("Unexpected exception during SSHManager initialization", e);
+        }
+    }
+
+    private static void createThread() throws IOException {
+        sshThread = new Thread(() -> {
+            try {
+                sshServerSetup.start();
+            } catch (IOException e) {
+                AL.warn("IOException occurred while starting SSH server", e);
+            } catch (Exception e) {
+                AL.warn("Failed to start SSH server", e);
+            }
+        });
+    }
+
+    private static void createConsoleCaptureThread() {
+        consoleCaptureThread = new Thread(() -> {
+            capturer.start();
+            while (!Thread.currentThread().isInterrupted()) {
+                try {
+                    String newOutput = capturer.getNewOutput();
+                    if (!newOutput.isEmpty()) {
+                        SSHServerConsoleReceive.broadcastToAll(newOutput);
+                    }
+                    Thread.sleep(500);
+                } catch (InterruptedException e) {
+                    Thread.currentThread().interrupt();
+                }
+            }
+        });
+    }
+
+    public static synchronized boolean start() {
+        AL.info("Starting SSH Server...");
+        if (isRunning()) {
+            AL.info("SSH Server is already running!");
+            return true;
+        }
+        try {
+            if (sshConfig.enabled.asBoolean()) {
+                if (sshThread.getState() == Thread.State.NEW) {
+                    sshThread.start();
+                    consoleCaptureThread.start();
+                } else if (sshThread.getState() == Thread.State.TERMINATED) {
+                    createThread();
+                    createConsoleCaptureThread();
+                    sshThread.start();
+                    consoleCaptureThread.start();
+                }
+                return true;
```

Comment:
Not sure what's going on above, shouldn't it only be the code of the second case?

Reply:
i guess youre right, fixed